### ISSUE TITLE
Missing Focus on Inputs

### ISSUE THREAD

**Issue by tomjn** [liyasthomas/postwoman#279]
**Describe the bug**
When clicking on a text input or a dropdown, there is no focus state. By default browsers add an outline for accessibility purposes, but this seems to have been removed ( regression? )

**To Reproduce**
1. Click on any input field

**Expected behavior**

When you click on a form input, it should be clear which input has gained focus

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Firefox
 - Version 70

**Comment by tomjn** [comment]
The only indication at the moment is the flashing text caret. I've clicked on one of these fields in the screenshot, and used the trackpad to scroll, see if you can guess which one is focused:

<img width="871" alt="Screenshot 2019-11-07 at 01 57 43" src="https://user-images.githubusercontent.com/58855/68353599-01347980-0102-11ea-8952-ebe6951f4163.png">

**Comment by liyasthomas** [comment]
Will consider adding a focus state indication ASAP 👍
